hack/run-test-e2e-serial.sh

```diff
@@ -12,5 +26,37 @@ if [ -n "${E2E_SERIAL_FOCUS}" ]; then
 	FOCUS="-ginkgo.focus=${E2E_SERIAL_FOCUS}"
 fi
 
-echo "Running Serial, disruptive E2E Tests"
-${BIN_DIR}/e2e-serial.test ${NO_COLOR} --ginkgo.v --ginkgo.failFast --ginkgo.reportFile=/tmp/artifacts/e2e-serial --test.parallel=1 ${FOCUS}
+function setup() {
+	if [[ x"${SETUP}" != "xtrue" ]]; then
+		return 0
+	fi
+	echo "Running NRO install test suite"
+	${BIN_DIR}/e2e-install.test ${NO_COLOR} --ginkgo.v --ginkgo.failFast --ginkgo.reportFile=/tmp/artifacts/e2e-serial-install --ginkgo.focus='\[Install\] continuousIntegration'
+}
+
+function teardown() {
+	if [[ x"${TEARDOWN}" != "xtrue" ]]; then
```

Comment:
```xtrue```?

Reply:
old habits die hard: https://stackoverflow.com/questions/174119/why-do-shell-script-comparisons-often-use-xvar-xyes

Reply:
TIL

Reply:
in this case I'm actually misleading because we explicitely require bash (not any `sh` compatible shell) so the conditions which will make this idiom useful just can't trigger. Will remove in the next upload.